server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/draft/methods/integration/SpamAssassinModule.java

```diff
@@ -49,6 +53,10 @@ protected void configure() {
                 new MailetConfigurationOverride(
                     org.apache.james.transport.mailets.SpamAssassin.class,
                     spamAssassinMailetConfig()));
+
+        bind(ListenersConfiguration.class)
```

Comment:
I'm lost, why do you need that?

Reply:
I can't inject the raw listener (as an extension it requires an executionMode that can't be injected as a core mailboxListener) 

Anyway now that we enforced it was a plugin ( cf https://github.com/linagora/james-project/pull/3359#pullrequestreview-409033388 ) the right testing strategy for plugin is to specify them in specific test via their POJO configuration, isn't it?

Reply:
Can you put this piece of code into a commit with a proper explanation?

Explain why it's needed now, what is the problem, how you solved it.

Reply:
Done with a wonderful 10 line git commit comment ;-)

Reply:
the comment seems to be truncated

Reply:
Crap